mypy/semanal.py

```diff
@@ -1283,13 +1286,19 @@ def process_import_over_existing_name(self,
 
     def normalize_type_alias(self, node: SymbolTableNode,
                              ctx: Context) -> SymbolTableNode:
+        normalized = False
         if node.fullname in type_aliases:
```

Comment:
This is the only place I can find where frozenset is treated differently -- do you understand why?